torch/csrc/jit/codegen/cuda/scheduler/transpose.cpp

```diff
@@ -415,12 +437,6 @@ std::shared_ptr<TransposeParams> getTransposeHeuristics(
 
   auto params = std::make_shared<TransposeParams>("Transpose heuristics");
 
-  // If the problem size is small use small tile sizes.
-  if (n_elems < device_multiprocessor_count * kMaxTileSize * kMaxTileSize) {
-    params->tile_size1 = 8;
-    params->tile_size2 = 8;
-  }
```

Comment:
Why is this removed?

Reply:
Based on benchmark results, using small tile size actually hurt the performance, instead of improving it...